mPowerSDK/Dashboard/APHDashboardViewController.m

```diff
@@ -497,13 +516,42 @@ - (void)prepareData
                     [rowItems addObject:row];
                 }
                     break;
+                    
+                /*
```

Comment:
see above comment about tremor task and uncommenting; also, mood section shouldn't be commented-out.